profile_helper.sh

```diff
@@ -92,7 +92,7 @@ done;
 # If $BASE16_THEME is set, this has already been loaded. This guards
 # against a bug where this script is sourced two or more times.
 if [ -n "$BASE16_THEME" ]; then
-  return 1
+  exit
```

Comment:
```suggestion
  exit 1
```

Reply:
happy to do it :)

Reply:
is there a reason the return should be non-zero?

Reply:
I get an error when it is non-zero :)

Reply:
specifically in the helper script - the hooks, I agree should be non-zero

Reply:
Hmm, strange I'm also running zsh on linux but I'm not getting an error in the profile helper. You can remove the exit code in the profile_helper if you want to. Sorry for all the changes. I'm just responding to your comments you're updating the pr so fast!